### PR TITLE
Added updated string files

### PR DESCRIPTION
**Pull request by aparthibanpaypal**

### CODE REVIEW

**Review by pdamodarannair — APPROVED** [review]

File: Common/Sources/Localization/cs.lproj/Localizable.strings [hunk 1]
```diff
@@ -21,39 +21,40 @@
...
-"fee_percent_only_max_formatter" = "Poplatek %1$@%% (max.:%2$@%3$@)";
-"fee_percent_formatter" = "Poplatek %1$@%% (min.:%2$@%3$@, max.:%2$@%4$@)";
-"fee_mix_no_min_and_max_formatter" = "Poplatek %1$@%2$@ + %3$@%%";
-"fee_mix_only_min_formatter" = "Poplatek %1$@%2$@ + %3$@%% (min.:%1$@%4$@)";
-"fee_mix_only_max_formatter" = "Poplatek %1$@%2$@ + %3$@%% (max.:%1$@%4$@)";
-"fee_mix_formatter" = "Poplatek %1$@%2$@ + %3$@%% (min.:%1$@%4$@, max.:%1$@%5$@)";
-"fee_percent_no_min_and_max_formatter" = "Poplatek %1$@%%";
+"mobileAreYouSure" = "Odebrat tento způsob převodu?";
+"fee_flat_formatter" = "Poplatek %1$@%2$@";
+"fee_percent_only_min_formatter" = "Poplatek %1$@%% (min.:%2$@%3$@";
```
Comment (ddhar-hw) [inline Common/Sources/Localization/cs.lproj/Localizable.strings:44]: I think these are messed up? Its missing a closing bracket. @malaw-hw @SaiNarendran can you confirm?

File: Common/Sources/Localization/it.lproj/Localizable.strings [hunk 2]
```diff
@@ -33,27 +34,27 @@
...
 "fee_flat_formatter" = "Commissione %1$@%2$@";
-"fee_percent_only_min_formatter" = "Commissione %1$@%% (Min:%2$@%3$@)";
-"fee_percent_only_max_formatter" = "Commissione %1$@%% (Max:%2$@%3$@)";
-"fee_percent_formatter" = "Commissione %1$@%% (Min:%2$@%3$@, Max:%2$@%4$@)";
-"fee_mix_no_min_and_max_formatter" = "%1$@%2$@ + commissione %3$@%%";
-"fee_mix_only_min_formatter" = "%1$@%2$@ + commissione %3$@%% (Min:%1$@%4$@)";
-"fee_mix_only_max_formatter" = "%1$@%2$@ + commissione %3$@%% (Max:%1$@%4$@)";
-"fee_mix_formatter" = "%1$@%2$@ + commissione %3$@%% (Min:%1$@%4$@, Max:%1$@%5$@)";
-"fee_percent_no_min_and_max_formatter" = "Commissione %1$@%%";
+"fee_percent_only_min_formatter" = "Commissione %1$@%% (Min:%2$@%3$@";
```
Comment (ddhar-hw) [inline Common/Sources/Localization/it.lproj/Localizable.strings:44]: Missing brackets?

File: Common/Sources/Localization/pl.lproj/Localizable.strings [hunk 3]
```diff
@@ -38,22 +39,22 @@
...
 "fee_flat_formatter" = "%1$@%2$@ opłata";
-"fee_percent_only_min_formatter" = "Opłata %1$@%% (min.: %2$@%3$@)";
-"fee_percent_only_max_formatter" = "Opłata %1$@%% (maks.: %2$@%3$@)";
-"fee_percent_formatter" = "Opłata %1$@%% (min.: %2$@%3$@, maks.: %2$@%4$@)";
-"fee_mix_no_min_and_max_formatter" = "Opłata %1$@%2$@ + %3$@%%";
-"fee_mix_only_min_formatter" = "Opłata %1$@%2$@ + %3$@%% (min.: %1$@%4$@)";
-"fee_mix_only_max_formatter" = "Opłata %1$@%2$@ + %3$@%% (maks.: %1$@%4$@)";
-"fee_mix_formatter" = "Opłata %1$@%2$@ + %3$@%% (min.: %1$@%4$@, maks.: %1$@%5$@)";
+"fee_percent_only_min_formatter" = "Opłata %1$@%% (min.: %2$@%3$@";
+"fee_percent_only_max_formatter" = "Opłata %1$@%% (maks.: %2$@%3$@";
```
Comment (ddhar-hw) [inline Common/Sources/Localization/pl.lproj/Localizable.strings:45]: Same here. Missing brackets

File: Common/Sources/Localization/th.lproj/Localizable.strings [hunk 4]
```diff
@@ -38,22 +39,22 @@
...
+"fee_percent_only_max_formatter" = "%1$@%% ค่าธรรมเนียม (สูงสุด:%2$@%3$@";
+"fee_percent_formatter" = "%1$@%% ค่าธรรมเนียม (ต่ำสุด:%2$@%3$@ สูงสุด:%2$@%4$@";
 "fee_mix_no_min_and_max_formatter" = "%1$@%2$@ + %3$@%% ค่าธรรมเนียม";
-"fee_mix_only_min_formatter" = "%1$@%2$@ + %3$@%% ค่าธรรมเนียม (ต่ำสุด:%1$@%4$@)";
-"fee_mix_only_max_formatter" = "%1$@%2$@ + %3$@%% ค่าธรรมเนียม (สูงสุด:%1$@%4$@)";
-"fee_mix_formatter" = "%1$@%2$@ + %3$@%% ค่าธรรมเนียม (ต่ำสุด:%1$@%4$@ สูงสุด:%1$@%5$@)";
-"fee_percent_no_min_and_max_formatter" = "%1$@%% ค่าธรรมเนียม";
+"fee_mix_only_min_formatter" = "%1$@%2$@ + %3$@%% ค่าธรรมเนียม (ต่ำสุด:%1$@%4$@";
+"fee_mix_only_max_formatter" = "%1$@%2$@ + %3$@%% ค่าธรรมเนียม (สูงสุด:%1$@%4$@";
+"fee_mix_formatter" = "%1$@%2$@ + %3$@%% ค่าธรรมเนียม (ต่ำสุด:%1$@%4$@ สูงสุด:%1$@%5$@";
```
Comment (ddhar-hw) [inline Common/Sources/Localization/th.lproj/Localizable.strings:50]: Same.. missing brackets

**Review by ddhar-hw — COMMENTED** [review]
Bunch of translation errors

File: Common/Sources/Localization/cs.lproj/Localizable.strings (hunk 1 above)
Comment (aparthibanpaypal) [reply Common/Sources/Localization/cs.lproj/Localizable.strings]: Updated.

File: Common/Sources/Localization/fr.lproj/Localizable.strings [hunk 5]
```diff
@@ -21,39 +21,40 @@
...
 "mobileTransferMethodLabel" = "MÉTHODE DE TRANSFERT";
 "mobileCountryRegion" = "Pays/Région";
 "mobileCurrencyLabel" = "Devise";
-"mobileAreYouSure" = "Êtes-vous sûr(e) ?";
+"mobileAreYouSure" = "Supprimer cette méthode de transfert ?";
 "fee_flat_formatter" = "%1$@%2$@ frais";
-"fee_percent_only_min_formatter" = "%1$@%% frais (Min. :%2$@%3$@)";
-"fee_percent_only_max_formatter" = "%1$@%% frais (Max. :%2$@%3$@)";
-"fee_percent_formatter" = "%1$@%% frais (Min. :%2$@%3$@, Max. :%2$@%4$@)";
+"fee_percent_only_min_formatter" = "%1$@%% frais (Min. :%2$@%3$@";
```
[the remark this answers is not in the document]
Comment (aparthibanpaypal) [reply Common/Sources/Localization/fr.lproj/Localizable.strings]: Fixed

File: Common/Sources/Localization/el.lproj/Localizable.strings [hunk 6]
```diff
@@ -33,27 +34,27 @@
...
-"fee_percent_only_max_formatter" = "Τέλος %1$@%% (μέγιστο: %2$@ %3$@)";
-"fee_percent_formatter" = "Τέλος %1$@%% (ελάχιστο: %2$@ %3$@, μέγιστο: %2$@ %4$@)";
-"fee_mix_no_min_and_max_formatter" = "Τέλος %1$@ %2$@ + %3$@%%";
-"fee_mix_only_min_formatter" = "Τέλος %1$@ %2$@ + %3$@%% (ελάχιστο: %1$@ %4$@)";
-"fee_mix_only_max_formatter" = "Τέλος %1$@ %2$@ + %3$@%% (μέγιστο: %1$@ %4$@)";
-"fee_mix_formatter" = "Τέλος %1$@ %2$@ + %3$@%% (ελάχιστο: %1$@ %4$@, μέγιστο: %1$@ %5$@)";
-"fee_percent_no_min_and_max_formatter" = "Τέλος %1$@%%";
+"mobileAreYouSure" = "Κατάργηση αυτού του τρόπου μεταφοράς;";
+"fee_flat_formatter" = "Τέλος %1$@%2$@";
+"fee_percent_only_min_formatter" = "Τέλος %1$@%% (ελάχιστο: %2$@%3$@";
```
[the remark this answers is not in the document]
Comment (aparthibanpaypal) [reply Common/Sources/Localization/el.lproj/Localizable.strings]: Fixed

File: Common/Sources/Localization/id.lproj/Localizable.strings [hunk 7]
```diff
@@ -48,31 +48,31 @@
 "fee_mix_only_min_formatter" = "Biaya %1$@%2$@ + %3$@%% (Min:%1$@%4$@)";
 "fee_mix_only_max_formatter" = "Biaya %1$@%2$@ + %3$@%% (Maks:%1$@%4$@)";
 "fee_mix_formatter" = "Biaya %1$@%2$@ + %3$@%% (Min:%1$@%4$@, Maks:%1$@%5$@)";
-"fee_percent_no_min_and_max_formatter" = "Biaya %1$@%%";
+"fee_percent_no_min_and_max_formatter" = "Biaya %1$@%% ";
```
[the remark this answers is not in the document]
Comment (aparthibanpaypal) [reply Common/Sources/Localization/id.lproj/Localizable.strings]: Fixed

File: Common/Sources/Localization/pl.lproj/Localizable.strings (hunk 3 above)
Comment (aparthibanpaypal) [reply Common/Sources/Localization/pl.lproj/Localizable.strings]: Fixed

File: Common/Sources/Localization/it.lproj/Localizable.strings (hunk 2 above)
Comment (aparthibanpaypal) [reply Common/Sources/Localization/it.lproj/Localizable.strings]: Fixed

File: Common/Sources/Localization/th.lproj/Localizable.strings (hunk 4 above)
Comment (aparthibanpaypal) [reply Common/Sources/Localization/th.lproj/Localizable.strings]: Fixed

**Review by ddhar-hw — APPROVED** [review]

**Pull request merged**